aries-site/src/components/accessibility/Video.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Box } from 'grommet';
+
+export const AccessVideo = () => {
+
+  return (
+    <Box width="large" pad={{ bottom: 'large' }}>
+      <video controls>
```

Comment:
Should we be using the `transcript` attribute and providing a transcription html file?

https://www.w3.org/WAI/PF/HTML/wiki/Media_Alt_Technologies#3:_Use_.40transcript_for_providing_a_link_to_a_file_with_the_full_transcription

Reply:
yes! will be adding this today

Reply:
just a heads up, i added the transcript file via a clickable link after the video that will open the html page. bill says this aligns with current best practice as a user will could save this page as a pdf if they wanted to digest the transcript offline.

i don't think the `transcript` attribute is implemented in HTML5 but is just an idea at the moment for future improvements to the `<video />`